frontend/public/components/utils/number-spinner.tsx

```diff
@@ -22,7 +22,9 @@ export const NumberSpinner: React.FC<NumberSpinnerProps> = ({
         inputProps={{ ...inputProps }}
         className={className}
         minusBtnAriaLabel={t('public~Decrement')}
+        minusBtnProps={{ 'data-test-id': 'Decrement' } as ButtonProps}
```

Comment:
```suggestion
        minusBtnProps={{ 'data-test': 'Decrement' } as ButtonProps}
```

Reply:
In most cased we use `data-test-id` and also for integration tests.